cmd/context/show.go

```diff
@@ -43,6 +44,18 @@ func Show() *cobra.Command {
 			if err := validateOutput(output); err != nil {
 				return err
 			}
+
+			// store the token value to display to user if the flag is set
+			TokenValue := current.Token
+
+			// By default, set the token value to null since default behaviour doesn't require to show token value
+			current.Token = ""
+
+			// If the flag is set, display the token value
+			if includeToken {
+				current.Token = TokenValue
+			}
+
```

Comment:
Why not simplify like this?

```suggestion
			if !includeToken {
				current.Token = ""
			}

```

Reply:
I would use this. Looks clearer

Reply:
hey @nitishfy, can you apply this change? wdyt?

Reply:
Resolved